### PR TITLE
Completed Issue #3246

### PR DESCRIPTION
**Pull request by LuD1161**
Implementations :
**Attendees CSV file :**
1-attendees_csv

**Attendees PDF File :**
2-attendees-pdf

**Attendees Export Button :**
3-attendees-button

**Orders Export Button :**
4-orders-button

**Orders CSV File :**
5-order-csv

**Orders PDF File :**
6-order-pdf

### CODE REVIEW

File: app/views/users/ticket_sales.py [hunk 1]
```diff
@@ -134,13 +145,13 @@ def display_ticket_stats(event_id):
...
-def display_orders(event_id):
+def display_orders(event_id, pdf=None):
     from_date = request.args.get('from_date')
     to_date = request.args.get('to_date')
     discount_code = request.args.get('discount_code')
     if ('from_date' in request.args and not from_date) or ('to_date' in request.args and not to_date) or \
-        ('from_date' in request.args and 'to_date' not in request.args) or \
-        ('to_date' in request.args and 'from_date' not in request.args):
+            ('from_date' in request.args and 'to_date' not in request.args) or \
+            ('to_date' in request.args and 'from_date' not in request.args):
```
Comment (rctay) [inline app/views/users/ticket_sales.py]: These whitespace changes are unneeded and should be omitted.

File: app/views/users/ticket_sales.py [hunk 2]
```diff
@@ -201,7 +215,7 @@ def display_attendees(event_id):
                 'firstname': holder.firstname,
                 'lastname': holder.lastname,
                 'email': holder.email,
-                'ticket_price': holder.ticket.price
+                'ticket_price': holder.ticket.price,
```
Comment (rctay) [inline app/views/users/ticket_sales.py]: This and the other addition of `,` are unnecessary and should be omitted.

File: app/views/users/ticket_sales.py [hunk 3]
```diff
@@ -243,9 +257,147 @@ def display_attendees(event_id):
...
+    value = 'Order#,Order Date, Status, First Name, Last Name, Email, Payment Type,' \
+            'Ticket Price \n'
+
+    for holder in holders:
+        if holder['status'] != "deleted":
+            # To avoid KeyError exception
+            try:
+                value += holder['order_invoice'] + ','
+            except:
+                value += ','
```
Comment (rctay) [inline app/views/users/ticket_sales.py]: Would this be better?

```
if 'order_invoice' in holder:
    value += holder['order_invoice']
value += ','
```

**Comment by rctay** [comment]
@LuD1161 As there is a need for this feature, I have merged your code with the corrections pointed out above and added a commit message for you. You can take the time to have a look for your learning, ping us if you have any questions :)

**Comment by mariobehling** [comment]
@rctay 

> I have merged your code with the corrections pointed out 

You have closed this PR without merging. Am I missing sth.?

**Pull request closed without merging**